code/modules/spells/spell_types/invoked/acolyte/noc.dm

```diff
@@ -11,12 +11,13 @@
 	req_items = list(/obj/item/clothing/neck/psycross/noc)
 	invocation = "Noc blinds thee of thy sins!"
 	invocation_type = "shout" //can be none, whisper, emote and shout
-	associated_skill = /datum/skill/magic/holy
 	antimagic_allowed = TRUE
 	recharge_time = 2 MINUTES
+	associated_skill = /datum/skill/magic/holy
+	miracle = TRUE
 	devotion_cost = 30
 
-/obj/effect/proc_holder/spell/invoked/blindness/cast(list/targets, mob/user = usr)
+/obj/effect/proc_holder/spell/invoked/blindness/miracle/cast(list/targets, mob/user = usr)
```

Comment:
Remove this

Reply:
Right, I re-added this because I thought it caused a bug earlier into this pr, mb.